research-log.md

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
Can we have some list of all the research log entries?

Reply:
Yes it is enabled 
https://staging.vac.dev/research